dencam/mppt.py

```diff
@@ -63,20 +74,20 @@ def log_solar_info():
                       'N/A', 'N/A', 'N/A',
                       'USB PORT ERROR']
     if not usb_error:
-        SunSaver.serial.baudrate = 9600
-        SunSaver.serial.stopbits = 2
+        sunsaver.serial.baudrate = 9600
+        sunsaver.serial.stopbits = 2
         try:
-            volt_batt = SunSaver.read_register(8) * 100 * 2**-(15)
-            volt_arr = SunSaver.read_register(9) * 100 * 2**-(15)
-            volt_ld = SunSaver.read_register(10) * 100 * 2**-(15)
-            curr_chrg = SunSaver.read_register(11) * 79.16 * 2**-(15)
-            curr_ld = SunSaver.read_register(12) * 79.16 * 2**-(15)
-            temp_amb = SunSaver.read_register(15)
-            temp_rts = SunSaver.read_register(16)
-            charge_state = SunSaver.read_register(17)
-            ah_charge = SunSaver.read_register(45) * 0.1
-            ah_load = SunSaver.read_register(46) * 0.1
-            alarm = SunSaver.read_register(50)
+            volt_batt = sunsaver.read_register(8) * 100 * 2**-(15)
+            volt_arr = sunsaver.read_register(9) * 100 * 2**-(15)
+            volt_ld = sunsaver.read_register(10) * 100 * 2**-(15)
+            curr_chrg = sunsaver.read_register(11) * 79.16 * 2**-(15)
+            curr_ld = sunsaver.read_register(12) * 79.16 * 2**-(15)
+            temp_amb = sunsaver.read_register(15)
+            temp_rts = sunsaver.read_register(16)
+            charge_state = sunsaver.read_register(17)
+            ah_charge = sunsaver.read_register(45) * 0.1
+            ah_load = sunsaver.read_register(46) * 0.1
+            alarm = sunsaver.read_register(50)
```

Comment:
You already know this (and we are saving this for a different issue/PR) but we should de-mystify all these magic numbers in the code.